### PR TITLE
Update HUB to gcr.io/istio

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Signed-off-by: JimmyCYJ <jimmychen.0102@gmail.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: istio/ci:go1.10-bazel0.15-clang6.0
+      - image: gcr.io/istio/ci:go1.10-bazel0.15-clang6.0
```
Comment (rkpagadala) [inline .circleci/config.yml]: should be
gcr.io/istio-testing/ci:go1.10-bazel0.15-clang6.0

Comment (qiwzhang) [reply .circleci/config.yml]: @rshriram It seems that we could not locate any Google people with the access.  Who owns the docker hub with admin permission to update the access list?  We like to add some Google test-infra people to the list too. I like to have access too as istio/proxy maintainer.

Comment (JimmyCYJ) [reply .circleci/config.yml]: @rshriram I was trying to update Envoy SHA, and found some build errors in circleCI [PR](https://github.com/istio/proxy/pull/1993), so I ask around and learn that I need to update circleCI image [PR](https://github.com/istio/proxy/pull/1994). We don't have write permission to the docker hub, could you add @qiwzhang, me and test-infra people to the access list? Is there any instruction to build images for circleCI?

**Comment by JimmyCYJ** [comment]
Thanks a lot @mandarjog, for helping us to build and push the images.

**Pull request closed without merging**